src/style.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.searchBar {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.searchBarBig {
+  width: 100%;
+  height: 70px;
+}
+
+.searchBarSmall {
+  width: 100%;
+  height: 42px;
+}
+
+.label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.label:after {
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.labelBig:after {
+  background-image: url("images/Search.svg");
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.labelSmall:after {
+  background-image: url("images/Search.svg");
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  font-weight: 300;
+  font-family: inherit;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inputBig {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.inputSmall {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.input::placeholder {
+  color: inherit;
+  font-weight: 300;
+}
+
+.inputBig::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.inputSmall::placeholder {
+  font-size: 14px;
+}
```

Comment:
pay attention to comments, you repeat yourself here, you've already set these styles above
```suggestion
```